buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1485,9 +1497,14 @@ class BeamModulePlugin implements Plugin<Project> {
     project.ext.applyPortabilityNature = {
       PortabilityNatureConfiguration configuration = it ? it as PortabilityNatureConfiguration : new PortabilityNatureConfiguration()
 
+      if (configuration.archivesBaseName) {
```

Comment:
nit: Shouldn't this be automatically handled by applyJavaNature already?

Reply:
Yes, it is redundant here. I wanted to localize this line to where the PortabilityConfiguration is processed. My thinking was to keep it somewhat independent of the behavior of applyJavaNature, or the fact that applyPortabilityNature depends on applyJavaNature. So the rest of either of those methods can change but this will not regress.